### ISSUE TITLE
Building xalienfs failing if libperl is missing

### ISSUE THREAD

**Issue by matthiasrichter** [alisw/alidist#392]
using alidist IB/v5-06/prod on ubuntu 14.04

I found a small glitch when building AliRoot. libperl is required for xalienfs and I had to install the libperl-dev.

Maybe that needs to be added dependency in the recipe.

Cheers
Matthias

**Comment by ktf** [comment]
The check is actually there:

```
  printf "#include <EXTERN.h>\n#include <perl.h>\n" | gcc -xc++ -I`perl -MConfig -e 'print $Config{archlib}'`/CORE - -c -o /dev/null && perl -MExtUtils::Embed -e 1
```

however on ubuntu it passes, for some reason I do not quite understand. If you have suggestions how to improve it, they are welcome.

**Comment by ktf** [comment]
Now fixed by #452 .
